### DISCUSSION TITLE
Solver 24.3 cannot be downloaded?

### DISCUSSION THREAD

**Discussion by DirkDali** [nextfoam/baram#100] [General]
Hello,
i can't download the new solver 24.3?

~$ wget https://d3c6e16xufx1gb.cloudfront.net/solvers_linux_24.3.0_20240726.tar.xz
--2024-07-31 21:17:01--  https://d3c6e16xufx1gb.cloudfront.net/solvers_linux_24.3.0_20240726.tar.xz
Auflösen des Hostnamens d3c6e16xufx1gb.cloudfront.net (d3c6e16xufx1gb.cloudfront.net) … 2600:9000:2093:5200:1:9af:26c0:21, 2600:9000:2093:d200:1:9af:26c0:21, 2600:9000:2093:1400:1:9af:26c0:21, ...
Verbindungsaufbau zu d3c6e16xufx1gb.cloudfront.net (d3c6e16xufx1gb.cloudfront.net)|2600:9000:2093:5200:1:9af:26c0:21|:443 … verbunden.
HTTP-Anforderung gesendet, auf Antwort wird gewartet … 403 Forbidden
2024-07-31 21:17:02 FEHLER 403: Forbidden.

**Comment by jiban** [comment]
Hi Dirk,
Sorry for the inconvenience.
There was an error for handling the file.
Now it's working.
By the way, please note that v24.3.0 has an issue related to cell zone configuration.
If you need cell zone configuration, please wait for the next patch release.
Thank you.

Best regards,
Jake
